plugin/builtin/buildbaron/build_baron.go

```diff
@@ -154,14 +154,14 @@ func (bbp *BuildBaronPlugin) buildFailuresSearch(w http.ResponseWriter, r *http.
 		bbp.opts.Password,
 	)
 
-	tickets, err := jiraHandler.JQLSearch(jql, 0, -1)
+	results, err := jiraHandler.JQLSearch(jql, 0, -1)
 	if err != nil {
 		message := fmt.Sprintf("%v: %v, %v", JIRAFailure, err, jql)
 		grip.Error(message)
 		plugin.WriteJSON(w, http.StatusInternalServerError, message)
 		return
 	}
-	plugin.WriteJSON(w, http.StatusOK, tickets)
+	plugin.WriteJSON(w, http.StatusOK, results.Issues)
```

Comment:
There's a comment from Sam in the ticket about linking to a Jira query instead of doing the query ourselves. Should we concerned about this Jira query consuming too many resources?